flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "automerge-flathubbot-prs": true,
+    "only-arches": ["x86_64"]
```

Comment:
:-(
any justification for restricting architectures?

Reply:
It was left over from the repo I used as a template. It's removed now.